.teamcity/_self/projects/WebApp.kt

```diff
@@ -123,6 +131,10 @@ object BuildDockerImage : BuildType({
 					--build-arg node_memory=32768
 					--build-arg use_cache=true
 					--build-arg base_image=%base_image%
+					--build-arg commit_sha=${Settings.WpCalypso.paramRefs.buildVcsNumber}
+					--build-arg manual_sentry_release=%MANUAL_SENTRY_RELEASE%
+					--build-arg is_default_branch=%teamcity.build.branch.is_default%
```

Comment:
I tried to combine the "manual release" and "is default branch" into the same variable, like "should_create_release", but it didn't seem like TeamCity was willing to process it correctly. For example, the following does not work:

```sh
--build-arg create_sentry_release="${ "%MANUAL_SENTRY_RELEASE%" === "true" || "%teamcity.build.branch.is_default%" === "true" }"
```

always resulted in `create_sentry_release="false"`. While the above syntax _does_ work in a Kotlin playground, I suspect that TeamCity isn't substituting the parameter values in this context, so they the statement is always false. The only way to work around that (that I could find) is to pass both values into Docker, and then Docker can make them available in the environment, and then webpack can access them on process.env and run a similar conditional itself.

Reply:
`is_default` === `trunk`?

Reply:
I think the `is_default` variable is just "true" or "false", but it's only true if we're on trunk